### ISSUE TITLE
Add "remove-orphans" option to "docker stack deploy"

### ISSUE THREAD

**Issue by lukewpatterson** [moby/moby#30976]
**Description**

Same purpose/behavior as `docker-compose up`'s [`--remove-orphans` option](https://github.com/docker/docker.github.io/blob/20c29f7e47ade7567ee35f3587790f6235d17d59/docs/reference/up.md).  

```
    --remove-orphans           Remove containers for services not
                               defined in the Compose file
```

I'd like to ensure that my compose file's representation of my stack's services is complete.

**Output of `docker version`:**

```
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Comment by lukewpatterson** [comment]
verified fixed with merged #31302

**Comment by thaJeztah** [comment]
Ah, yes, this is implemented as `--prune` in docker 17.05 and up 👍  I'll close this one, thanks!
